sdk/onlineexperimentation/azure-analytics-onlineexperimentation/README.md

```diff
@@ -0,0 +1,78 @@
+# Azure OnlineExperimentation client library for Java
+
+Azure OnlineExperimentation client library for Java.
+
+This package contains Microsoft Azure OnlineExperimentation client library.
+
+## Documentation
+
+Various documentation is available to help you get started
+
+- [API reference documentation][docs]
+- [Product documentation][product_documentation]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-analytics-onlineexperimentation;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-analytics-onlineexperimentation</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Authentication
+
+[Azure Identity][azure_identity] package provides the default implementation for authenticating the client.
+
+## Key concepts
+
+## Examples
+
+```java com.azure.analytics.onlineexperimentation.readme
+```
```

Comment:
Please have at least 1 sample in your README.

You can use any of one in your current samples.